### PR TITLE
Create nullable-cupertinothemedata-brightness.md

### PR DESCRIPTION
**Pull request by LongCatIsLooong**

### CODE REVIEW

**Comment by sfshaza2** [comment]
Hi Long Cats! I've merged the outstanding PR.  Can you fix the conflicts?

File: src/docs/release/breaking-changes/index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ available:
 * [TestTextInput state reset]
 * [TextInputClient currentTextEditingValue]
 * [ThemeData's accent properties]
+* [Nullable `CupertinoTheme.brightness`]
```
Comment (goderbauer) [inline src/docs/release/breaking-changes/index.md]: nit: this is not in alphabetical order (see text above the list).

**Comment by goderbauer** [comment]
> LGTM! But, @goderbauer just merged some changes. Can you fix the conflict?

Sorry!

**Comment by sfshaza2** [comment]
True. It should be alphabetized, but I can fix it later.

File: src/docs/release/breaking-changes/index.md [hunk 2]
```diff
@@ -11,17 +11,17 @@ available:
...
 * [Scrollable AlertDialog]
 * [TestTextInput state reset]
 * [TextInputClient currentTextEditingValue]
 * [ThemeData's accent properties]
-* [Nullable `CupertinoTheme.brightness`]
 
 [breaking change policy]: /docs/resources/compatibility
+[Generic type of ParentDataWidget changed to ParentData]: /docs/release/breaking-changes/parent-data-widget-generic-type
 [ImageCache large images]: /docs/release/breaking-changes/imagecache-large-images
+[Nullable `CupertinoTheme.brightness`]: /docs/release/breaking-changes/nullable-cupertinothemedata-brightness.md
```
Comment (sfshaza2) [inline src/docs/release/breaking-changes/index.md]: D'OH! Please delete the ".md" from the end.

**Comment by sfshaza2** [comment]
It can be merged once the link error is fixed. (By removing the ".md" from the end of the URL.)

**Comment by sfshaza2** [comment]
It looks fixed to me!

**Pull request merged**